include/modem/pdn.h

```diff
@@ -54,6 +54,42 @@ struct pdn_pdp_opt {
 	uint8_t secure_pco;
 };
 
+/**
+ * @brief PDN PDP context dynamic information structure.
```

Comment:
The PDN library uses both `PDN` and `PDP` abbreviations, but `PDP` is legacy from before LTE and I'm thinking about cleaning the PDN library API. Anyway, please remove all uses of `PDP` from new implementation. `PDN` and `PDN context` are fine.

Reply:
Ok, done.